static/locales/fr/translation.json

```
@@ -7,26 +7,26 @@
 			"title": "Paramètres du lobby",
 			"gamehostonly": "Seul l'hôte de la partie peut modifier cela!",
 			"inlobbyonly": "Vous ne pouvez changer cela que dans le lobby!",
-			"voicedistance": "Distance vocal",
-			"voicedistance_impostor": "Distance vocal imposteurs",
+			"voicedistance": "Distance vocale",
+			"voicedistance_impostor": "Distance vocale imposteurs",
 			"wallsblockaudio": "Les murs bloquent l'audio",
```

```suggestion
			"wallsblockaudio": "Les murs bloquent le son",
```
It sounds quite more natural in French to me, WDYT?